yare-model-validators/src/main/java/com/sabre/oss/yare/model/validator/ReferenceValidator.java

```diff
@@ -115,10 +118,12 @@ private void checkNamesAreNotDuplicated(List<String> names, ValidationResults re
         }
     }
 
-    private void checkReservedNamesAreNotUsed(List<String> names, ValidationResults results) {
-        if (names.contains(CONTEXT)) {
-            append(results, ValidationResult.error("rule.ref.reserved-names",
-                    String.format("Naming Error: Reserved names are used -> [%s]", CONTEXT)));
+    private void checkReservedNamesAreNotUsed(List<String> names, ValidationResults results, String... reservedNames) {
```

Comment:
This vararg here is not needed - method is private and executed once. I would rather put reservedNames in method body. Maybe stream (`Stream.of(CONTEXT, ENGINE_CONTROLLER)...`) ?

Reply:
Done.